server/views/pages/reviewer/view-report.html

```diff
@@ -13,7 +13,7 @@ <h1 class="govuk-heading-xl">{{ pageTitle }}</h1>
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
         {{
-          reportDetail.reportHeading(data)
+          reportDetail.reportHeading(data, false)
```

Comment:
inconsistent in how you are calling this macro

maybe `reportDetail.reportHeading(data, print=false)` would be better?

Reply:
done